src/NuGet.Protocol.Core.v3/DataClient/RetryHandler.cs

```diff
@@ -26,28 +31,37 @@ public RetryHandler(HttpMessageHandler innerHandler, int retries)
 
             var success = false;
 
-            while (tries < _retries
+            while (tries < MaxTries
                    && !success)
             {
-                // wait progressively longer
+                // wait before trying again if the last call failed
                 if (!success
                     && tries > 0)
                 {
-                    await Task.Delay(500 * tries, cancellationToken);
+                    await Task.Delay(200, cancellationToken);
```

Comment:
Good catch, RetryDelay was just added for the tests. I'll get this hooked into the actual code.